x-pack/plugins/security_solution/public/common/components/events_viewer/index.tsx

```diff
@@ -212,6 +212,7 @@ const StatefulEventsViewerComponent: React.FC<Props> = ({
               dataProviders,
               dataViewId,
               defaultCellActions,
+              visibleCellActions: 3,
```

Comment:
@MikePaquette (pinging you because you posted in https://github.com/elastic/eui/pull/5675#issuecomment-1062902133)

I'm not overly familiar with Security's usage of EuiDataGrid - can you confirm that this component (Events Viewer?) is the correct/only component that needs 3 visible cell actions instead of the default 2?

Reply:
From a slack thread:

> imo would be safer to just pass a hard coded 3 here https://github.com/elastic/kibana/blob/main/x-pack/plugins/timelines/public/components/t_grid/body/index.tsx#L819-#L839 instead of as a prop from the different places it’s used

Reply:
> @MikePaquette (pinging you because you posted in [elastic/eui#5675 (comment)](https://github.com/elastic/eui/pull/5675#issuecomment-1062902133))
> 
> I'm not overly familiar with Security's usage of EuiDataGrid - can you confirm that this component (Events Viewer?) is the correct/only component that needs 3 visible cell actions instead of the default 2?

@constancecchen yes, this is the only component that needs 3 visible cell actions instead of the default 2